core/src/mindustry/world/Tiles.java

```diff
@@ -7,18 +7,40 @@
 
 import java.util.*;
 
-/** A tile container. */
+/**
+ * Contains a list of tiles with extended behavior
```

Comment:
This class should use one line whenever possible, like `Block` does:
```java
/** doc 1 */
int field1;
/** doc 2 */
int field2;
```

Extra newlines waste a lot of space and make things harder to read.